pkgs/development/tools/java/jprofiler/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, lib, fetchurl, makeWrapper, makeDesktopItem, jdk }:
+
+stdenv.mkDerivation rec {
+  version = "13.0.2";
+  pname = "jprofiler";
+
+  src = fetchurl {
+    url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_linux_${builtins.replaceStrings ["."] ["_"]  version}.tar.gz";
+    sha256 = "sha256-x9I7l2ctquCqUymtlQpFXE6+u0Yg773qE6MvAxvCaEE=";
+  };
+
+  srcIcon = fetchurl {
+    url = "https://www.ej-technologies.com/assets/content/header-product-jprofiler@2x-24bc4d84bd2a4eb641a5c8531758ff7c.png";
+    sha256 = "sha256-XUmuqhnNv7mZ3Gb4A0HLSlfiJd5xbCExVsw3hmXHeVE=";
+  };
+
+  desktopItem = makeDesktopItem {
+      name = pname;
+      exec = pname;
+      icon = pname;
+      comment = meta.description;
+      desktopName = "JProfiler";
+      genericName = "Java Profiler Tool";
+      categories = [ "Development" ];
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    cp -r . $out
+
+    wrapProgram $out/bin/jpanalyze --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jprofiler --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpcompare --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpcontroller --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpenable --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpexport --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpintegrate --set JAVA_HOME "${jdk.home}"
+    wrapProgram $out/bin/jpenable --set JAVA_HOME "${jdk.home}"
```

Comment:
```suggestion
    for i in $out/bin/*; then
      wrapProgram $i --set JAVA_HOME "${jdk.home}" 
    fi
```

Reply:
I think this broke the build because it's supposed to be 
```
    for i in $out/bin/*; do
      wrapProgram $i --set JAVA_HOME "${jdk.home}" 
    done
```
My bad.

Reply:
@azahi and I've missed that. Thanks for fixing it.